hs_modelinstance/tests/test_modelinstance_metadata.py

```diff
@@ -49,12 +49,19 @@ def setUp(self):
         )
 
         self.temp_dir = tempfile.mkdtemp()
+
         self.file_name = "MIR.txt"
         temp_text_file = os.path.join(self.temp_dir, self.file_name)
         text_file = open(temp_text_file, 'w')
         text_file.write("Model Instance resource files")
         self.text_file_obj = open(temp_text_file, 'r')
 
+        self.file_name_2 = "MIR2.txt"

```

Comment:
I would change the extension of this file as the unit test that is using it trying to test that any file types can be added to the resource.